cartoframes/data/observatory/catalog.py

```diff
@@ -40,3 +40,20 @@ def datasets(self):
         """
 
         return Dataset.get_all()
+
+    @classmethod
+    def purchased_datasets(self, credentials=None):
+        """Get all the datasets in the Catalog
+
+        Args:
+            credentials (:py:class:`Credentials <cartoframes.auth.Credentials>`, optional):
+                A :py:class:`Credentials <cartoframes.auth.Credentials>`
+                instance can be used in place of a `username`|`base_url` / `api_key` combination.
+                Only required for the purchased datasets.
+
+        Returns:
+            :py:class:`Datasets <cartoframes.data.observatory.Datasets>`
+
+        """
+
+        return Dataset.get_all(credentials)
```

Comment:
What does it happen in discovery If the user has used `set_default_credentials`? i think we are using the same method for both cases (discovery and purchased) and in the discovery we are going to return only the purchased ones.